core/modules/css/CssAccess.cc

```diff
@@ -503,7 +503,8 @@ CssAccess::getPartTableParams(std::string const& dbName,
 
     std::vector<std::string> subKeys{"partitioning", "partitioning/subChunks",
         "partitioning/dirDb", "partitioning/dirTable", "partitioning/dirColName", "partitioning/latColName",
-        "partitioning/lonColName", "partitioning/overlap", "partitioning/secIndexColName"};
+        "partitioning/lonColName", "partitioning/overlap", "partitioning/secIndexColName",
+        "partitioning/lockInMem", "partitioning/scanSpeed"};
```

Comment:
I still think it's better to save them in two separate groups in KV. Partitioning parameters are supposed to be stable, but you may want to modify shared scan parameters. It would be easier to implement updating is shared scan parameters are in their own group. And I'm still a bit unsure that adding shared scan parameters to `PartTableParams` struct is the way to go, we may want to rethink it in the future.